docs/devsite-help/troubleshooting.md

```diff
@@ -141,6 +141,35 @@ In that case, the simplest fix is to add a direct dependency to
 `Grpc.Core` from your application, which will ensure that the
 native libraries are copied appropriately.
 
+## How to get ErrorInfo from SpannerException
+
+The ErrorInfo message provides error information that both humans and
+applications can depend on. If SpannerException contains RPCException
+as it's inner execption, we can extract ErrorInfo from SpannerException.
```

Comment:
```suggestion
as its inner exception, we can extract ErrorInfo from SpannerException.
```

Reply:
I'd suggest also using backticks for `ErrorInfo` and `SpannerException`